index.html

```diff
@@ -27,9 +27,32 @@ <h3>
                     Welcome
                 </h3>
             </a>
-            <nav>
+            <button type="button" class="menu-btn">
```

Comment:
Kindly provide the requested design on Figma:
![Screen Shot 2021-12-13 at 21 28 52](https://user-images.githubusercontent.com/22258274/145927963-b0e35bac-9e99-4858-922b-48ae877fb4bb.png)


#

**_Yours:_**
![Screen Shot 2021-12-13 at 21 27 12](https://user-images.githubusercontent.com/22258274/145927846-67e6d184-d03f-42fa-bbab-4f54f0d8ced6.png)

Your `menu` should cover the body.